fontbe/src/features.rs

```diff
@@ -158,7 +158,15 @@ pub(crate) fn resolve_variable_metric<'a>(
 ) -> Result<(i16, Vec<(VariationRegion, i16)>), DeltaError> {
     let point_seqs: HashMap<_, _> = values
         .into_iter()
-        .map(|(pos, value)| (pos.to_owned(), vec![value.0 as f64]))
+        .map(|(pos, value)| {
+            // The master values for anchor positions or kerning adjustments are
+            // expected to be rounded before computing the deltas, because instancing
+            // a VF at the masters' location is expected to be equivalent to building
+            // individual masters as static fonts. fontmake does the same, see
+            // https://github.com/googlefonts/fontc/issues/1043
+            let value: f32 = value.into_inner().ot_round();
```

Comment:
should we ot_round to f64 and avoid f32 then f64? - I recall we need to add that to otround, maybe we should do that first?

Reply:
I already added OtRound<f64> for f64 and OtRound<f32> for f32 here: 
https://github.com/googlefonts/fontations/pull/1209 (unreleased yet, which is why this PR is fetching fontations from github)

Are you saying we should also add OtRound<f64> for f32? Then why not any other combinations of primitive number types? It feels to me a bit out of scope for a rounding trait that we are casting f32 as f64 because of some other API which works best with those.